### PR TITLE
mgr/PGMap: drop REQUEST_{SLOW,STUCK} HEALTH_WARNs

### PR DESCRIPTION
**Pull request by tchaikov**
SLOW_OPS unifies both of them.

Signed-off-by: Kefu Chai <kchai@redhat.com>

### CODE REVIEW

**Comment by tchaikov** [comment]
it's a follow-up of #18614.

File: src/mon/PGMap.cc [hunk 1]
```diff
@@ -2621,7 +2621,9 @@ void PGMap::get_health_checks(
 
   // REQUEST_SLOW
   // REQUEST_STUCK
-  if (cct->_conf->mon_osd_warn_op_age > 0 &&
+  // SLOW_OPS unifies them in mimic.
+  if (osdmap.require_osd_release < CEPH_RELEASE_MIMIC &&
```
Comment (xiexingguo) [inline src/mon/PGMap.cc]: But below we still use REQUEST_SLOW/ REQUEST_STUCK as the warn-titles...

Comment (tchaikov) [reply src/mon/PGMap.cc]: @xiexingguo i don't follow you. if you need more context, please see #18614.

Comment (xiexingguo) [reply src/mon/PGMap.cc]: e.g.:
```c++
     if (!warn_detail.empty()) {
       ostringstream ss;
       ss << warn << " slow requests are blocked > "
 	 << cct->_conf->mon_osd_warn_op_age << " sec";
auto& d = checks->add("REQUEST_SLOW", HEALTH_WARN, ss.str());
```

We still use "REQUEST_SLOW" as the warn-type

Comment (tchaikov) [reply src/mon/PGMap.cc]: again, could you read John's comment in #18614 , where `SLOW_OPS` unifies these two warnings. so after mimic, they will vanish, but before that, pre-mimic OSDs should be able to report to mgr.

Comment (xiexingguo) [reply src/mon/PGMap.cc]: Ahh, right. Thanks for the explanation, I got it now

File: src/mon/PGMap.cc [hunk 2]
```diff
@@ -2621,7 +2621,9 @@ void PGMap::get_health_checks(
...
+      cct->_conf->mon_osd_warn_op_age > 0 &&
```
Comment (shinobu-x) [inline src/mon/PGMap.cc]: What about using `get_val<>` here?

Comment (tchaikov) [reply src/mon/PGMap.cc]: @shinobu-x that's not in the scope of this change.

**Comment by tchaikov** [comment]
- http://pulpito.ceph.com/kchai-2017-11-23_12:43:54-rados-wip-kefu-testing-2017-11-23-1812-distro-basic-mira/
- http://pulpito.ceph.com/kchai-2017-11-24_08:53:47-rados-wip-kefu-testing-2017-11-23-1812-distro-basic-mira/

**Comment by tchaikov** [comment]
@jcsp ping?

**Comment by tchaikov** [comment]
retest this please.

**Comment by tchaikov** [comment]
@jcsp ping?

**Comment by tchaikov** [comment]
@jcsp ping?

**Pull request merged**